src/Provider/Provider.Web/ViewModels/Part1/Employer/EmployersViewModel.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+using Esfa.Recruit.Shared.Web.ViewModels;
+
+namespace Esfa.Recruit.Provider.Web.ViewModels.Part1.Employer
+{
+    public class EmployersViewModel : EmployersEditViewModel
```

Comment:
Apparently we're not inheriting from the edit model any more so remove this dependency and copy over the required params.